### DISCUSSION TITLE
How to write raw values in v4

### DISCUSSION THREAD

**Discussion by Rory-Z** [mikefarah/yq#1491] [Q&A] [answered]
Hello, I'm using v4.30.6 in MacBook, and I want write raw value in my file.
I see this issue: https://github.com/mikefarah/yq/issues/107, it looks work in v3 and v2, but I don't know how to work in v4, it looks don't have `--raw` flag in v4
Please help me, thanks

**Comment by mikefarah** [comment, marked as answer]
v4 is like `jq` - so you can do something like:

```bash
yq '.a = "1000"' file.yaml
```

**Reply by Rory-Z** [reply]
Cool, thanks
